### DISCUSSION TITLE
display validation rules

### DISCUSSION THREAD

**Discussion by nataliiatytovych** [Laravel-Backpack/community-forum#1078] [Q&A (Help)]
Hi everyone!
```
class NewsRequest extends FormRequest
{
    public function rules(): array
    {
        return [
            'title' => 'required|max:150',
        ];
    }

    public function messages()
    {
        return [
            'title.required' => 'Tbjbkjk.',
        ];
    }
}
```

```
protected function setupCreateOperation(): void
    {
        $this->crud->setValidation(NewsRequest::class);
        ....
    }
```
    
   

this is my form request and validation in crud controller. Why all validation errors are not displayed in user friendly way? i got this error: "errors":{"code":"0001","message":{"title":["Tbjbkjk."]}}}

**Comment by karandatwani92** [comment]
Hi @nataliiatytovych 

I just tried the same and the custom error message is being displayed properly:

![Screenshot 2024-07-11 at 1 30 15 PM](https://github.com/Laravel-Backpack/community-forum/assets/8214221/2932e66d-57d6-4401-acdb-41079cc287f9)

**Reply by nataliiatytovych** [reply]
do you know classes or config files which i may change to debug? i have tried all 3 ways to set up validation and none of them works:(
